server/src/lib.rs

```diff
@@ -144,8 +146,20 @@ impl InitData {
                 run.storage.fs_path
             ))?;
         }
+
         let storage = DispatchBackend::Filesystem(FileSystemBackend::new(storage).await?);
 
+        // TODO: where/how should we configure these details?
+        let ui = UI {
+            version: env!("CARGO_PKG_VERSION").to_string(),
+            auth_required: String::from("false"),
+            oidc_server_url: String::from("http://localhost:8180/realms/trustify"),
+            oidc_client_id: String::from("trustify-ui"),
+            oidc_scope: String::from("email"),
```

Comment:
I believe it should be populated with the values of `AuthConfigArguments` https://github.com/trustification/trustify/blob/main/common/auth/src/auth.rs#L15

But we can do it in another PR

Reply:
Okay, I'll work with @ctron in a subsequent PR.